README.md

```diff
@@ -69,7 +69,7 @@ If you would like to make a change to the site:
 If you are a committer, do the following:
 
 1. Update the master branch with your (or a Pull Request's) change.
-2. Push updated master to the asf remote master (https://git-wip-us.apache.org/repos/asf/apex-site.git)
+2. Push updated master to the asf remote master (https://github.com/apache/apex-site.git)
```

Comment:
Refer to "Merging a Pull Request" or mentioned that for a PR it is possible to use "Rebase and merge" option on github.